internal/test/wmcb/wmcb_test.go

```diff
@@ -58,15 +59,18 @@ func TestWMCBUnit(t *testing.T) {
 	os.Stdout = w
 
 	// Remotely execute the test binary.
-	_, err = framework.WinrmClient.Run(remotePowerShellCmdPrefix+framework.RemoteDir+"\\"+
+	exitCode, err := framework.WinrmClient.Run(remotePowerShellCmdPrefix+framework.RemoteDir+"\\"+
 		"wmcb_unit_test.exe --test.v",
 		os.Stdout, os.Stderr)
 	assert.NoError(t, err, "error while executing the test binary remotely")
+	assert.Equal(t, 0, exitCode, "remote binary returned non-zero exit code")
 	w.Close()
 	out, err := ioutil.ReadAll(r)
 	assert.NoError(t, err, "error reading stdout from the remote Windows VM")
 	os.Stdout = stdout
+	log.Printf("%s", out)
```

Comment:
Is this left over from debugging? Otherwise looks good

Reply:
No. Please read the commit message: `To help with debugging we are now printing out the complete output of the remote test run.`

Reply:
Ah ok.
/lgtm